src/EnergyPlus/HVACUnitaryBypassVAV.hh

```diff
@@ -72,6 +72,7 @@ namespace HVACUnitaryBypassVAV {
     constexpr int Off(0); // Signal DXCoil that compressor should not run
 
     // Mode of operation
+    // can't change these to enum class since these are used in SetupOutputVariable()
```

Comment:
Yeah, another one here where we are reporting a discrete term to the time series output.  I would think we could just get this information from looking at actual floating point system heat transfer rates rather than having to report out an integer control signal.  For later...